packages/grafana-ui/src/components/Table/TableNG/TableNG.tsx

```diff
@@ -111,10 +111,14 @@ export function TableNG(props: TableNGProps) {
   } = props;
 
   const theme = useTheme2();
-  const styles = useStyles2(getGridStyles, {
-    enablePagination,
-    transparent,
-  });
+  const styles = useMemo(
+    () =>
+      useStyles2(getGridStyles, {
+        enablePagination,
+        transparent,
+      }),
+    [enablePagination, transparent]
+  );
```

Comment:
ok. so the issue here is that we're passing in an object instead of positional arguments, it seems: https://github.com/grafana/grafana/blob/fbca70747caad9845945fa0b4fcc2d1774a8d14e/packages/grafana-ui/src/themes/ThemeContext.tsx#L131

I think that the `memoize` method here expects you to pass a list of args into useStyles which can be compared for equality by reference. I'm just going to update this to pass these two booleans back-to-back. We used to take many more args into this method, but now it's not so bad to just list the two bools as positional args.